dydx/dydxPresenters/dydxPresenters/_v4/Vault/DepositsAndWithdrawals/dydxVaultDepositWithdrawConfirmationViewBuilder.swift

```diff
@@ -43,63 +46,138 @@ private class dydxVaultDepositWithdrawConfirmationViewController: HostingViewCon
 
 private protocol dydxVaultDepositWithdrawConfirmationViewPresenterProtocol: HostedViewPresenterProtocol {
     var viewModel: dydxVaultDepositWithdrawConfirmationViewModel? { get }
-    var transferType: VaultTransferType { get set }
+    var transferType: VaultTransferType? { get set }
+    var amount: Double? { get set }
```

Comment:
this is another result of moving input state out of abacus. We will have heavier presenter protocols with more properties which have to be initialized during the `arrive` step. The fact that these are nullable is also not great, easy to forget to initialize them.